web-components/src/components/inputs/new/FileDrop/FileDrop.tsx

```diff
@@ -156,11 +157,12 @@ const FileDrop = forwardRef<HTMLInputElement, FileDropProps>(
       });
 
       if (result) {
-        setValue(result, currentFile.type, fieldIndex);
+        setValue(result, currentFile.type, fieldIndex + nextFieldIndex);
```

Comment:
this was mostly already implemented but there was a small bug when setting the value that would replace the first uploaded photo with the next one